using-valuesets-in-rules.md

```diff
@@ -2,9 +2,27 @@
 
 ValueSets can be used by the CQL prepopulation logic. These ValueSets can come from the ruleset's local resources or from VSAC.
 
+## Referencing ValueSets
+
+When ValueSets are used, they should be referenced by their canonical URL in most cases.
+
+### VSAC ValueSets
+
+VSAC ValueSets should be referenced by the canonical URL for VSAC, `http://cts.nlm.nih.gov/fhir/ValueSet/{OID}`. For example, the VSAC ValueSet with OID `2.16.840.1.113762.1.4.1219.3` has the the canonical URL `http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.3`. This should be used in all places when referencing the ValueSet. CRD will load expansions automatically for these ValueSets when referenced assuming it has VSAC credentials available.
+
+### Locally Defined ValueSets
+
+Local ValueSets created specifically for a DTR rule can be stored in the `Shared` or topic's respective `resources` folder. It can be referenced by it's own id in CQL logic. But in the Library or Questionnaire resources it must use a faux canonical URL, `<server-path>ValueSet/{id}`. For example, a local ValueSet with id `copd` should be referenced by `copd` in the CQL valueset definition. In the FHIR Library or Questionnaire, it should be referened by the faux canonical URL. `<server-path>ValueSet/copd`. The CRD server will replace the `<server-path>` chunk with the URL of the FHIR endpoint on the CRD server.
```

Comment:
Do we still recommend locally defined valueset, e.g. copd? What is the use case?

Reply:
Definitely do not recommend using locally defined valuesets.